packages/circuit-ui/components/DateInput/DateInput.tsx

```diff
@@ -358,6 +370,23 @@ export const DateInput = forwardRef<HTMLDivElement, DateInputProps>(
             />
           </FieldLegend>
           <div ref={fieldRef} className={classes.wrapper}>
+            <input
+              type="date"
+              ref={applyMultipleRefs(ref, inputRef)}
+              className={classes.hidden}
+              min={min}
+              max={max}
+              required={required}
+              disabled={disabled}
+              readOnly={readOnly}
+              autoComplete={autoComplete}
+              aria-invalid={invalid}
```

Comment:
Technically, these attributes are superfluous given that the input is hidden, however, it's more correct to set them in case someone reads them from `event.target`.